examples/todomvc/test/todomvc/core_test.cljc

```diff
@@ -0,0 +1,126 @@
+(ns todomvc.core-test
+  (:require #?(:cljs [cljs.test :refer-macros [deftest is]]
+               :clj  [clojure.test :refer [deftest is]])
```

Comment:
Fwiw, from ClojureScript 1.9.198 onwards we can eliminate the reader conditional here and just use the `:clj` branch, and ClojureScript will figure it out.